simplebot/deltabot.py

```diff
@@ -46,19 +50,23 @@ def __init__(self, basedir, os_name=None):
         self.account.set_config('sentbox_watch', '0')
         self.account.set_config('mvbox_watch', '0')
         self.account.set_config('bcc_self', '0')
+        self.account.add_account_plugin(self)
+        self.account.add_account_plugin(FFIEventLogger(self.account, ""))
 
     def is_configured(self):
         return bool(self.account.is_configured())
 
     def configure(self, email, password):
-        self.account.configure(addr=email, mail_pw=password)
-        self.account.start_threads()
-        configured = self._wait_configuration_progress(1000) >= 1000
-        self.account.stop_threads()
-        if configured:
-            self.logger.info('Bot configured successfully!')
-        else:
-            self.logger.info('Configuration failed')
+        with self.account.temp_plugin(ConfigureTracker()) as configtracker:
+            self.account.start_threads()
```

Comment:
```suggestion
            self.account.start_threads()
            self.account.configure(addr=email, mail_pw=password)
```

`Account.start_threads()` and `Account.configure()` don't exist anymore, right? I guess this two lines should be replaced by `self.start()` now